includes/wikia/nirvana/WikiaView.class.php

```diff
@@ -114,36 +109,106 @@ protected function buildTemplatePath( $controllerClass, $methodName, $forceRebui
 			}
 
 			// Service classes must be dispatched by full name otherwise we default to a controller.
-			$controllerBaseName = $app->getBaseName( $controllerClass );
-			if ($app->isService($controllerClass)) {
-				$controllerClass = $app->getServiceClassName( $controllerBaseName );
-			} else {
-				$controllerClass = $app->getControllerClassName( $controllerBaseName );
+			$controllerClass = $this->normalizeControllerClass( $controllerClass );
+
+			$templateExists = false;
+			$templatePath = '';
```

Comment:
Hmm, not sure if I agree.  "Filename" could mean the full path or just the name portion of a path, so its a bit ambiguous.